packages/menu/src/MenuItem.ts

```diff
@@ -518,6 +626,19 @@ export class MenuItem extends LikeAnchor(
         this.updateAriaSelected();
     }
 
+    protected override willUpdate(changes: PropertyValues<this>): void {
```

Comment:
If focus is inside the submenu when it close, move focus back to its parent item.